script/predicate_functions.sh

```diff
@@ -23,18 +23,6 @@ function is_mri_192 {
   fi
 }
 
-function is_mri_192_plus {
-  if is_mri; then
-    if ruby -e "exit(RUBY_VERSION.to_f > 1.9)"; then
-      return 0
-    else
-      return 1
-    fi
-  else
-    return 1
-  fi
-}
-
```

Comment:
I see. Have never used those, so I'll the ones who have already "suffered" them... :)